cypress/e2e/undo-redo/undo-redo.cy.js

```diff
@@ -0,0 +1,27 @@
+describe('undo-redo', () => {
```

Comment:
I think it would be more specific if we name this "block-creation.cy.js" because it's under `undo-redo` folder. Besides, we plan to structure a bit the e2e folder to extra non undoRedo feature to be outside